components/grid.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { View, Text, StyleSheet, Image } from 'react-native';
+
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    padding: 10,
+    flexDirection: 'column',
+    alignItems: 'center',
+  },
+  text: {
+    fontSize: 20,
+    textAlign: 'left',
+    fontWeight: 'bold'
+  },
+  num: {
+    fontSize: 18,
+    textAlign: 'right'
+  },
+  photo: {
+    height: 350,
+    width: 350,
+    borderRadius: 5,
+  },
+});
+
+const Grid = (props) => (
```

Comment:
`GridItem`. But if you look in it, it's doing direct references to a challenge. `ChallengeGridItem` ?